src/database/config/passport.js

```diff
@@ -0,0 +1,56 @@
+import { Strategy } from "passport-google-oauth20";
```

Comment:
Now for this config, you don't need to add it to database -> config again, just add it to config-> config. that is where it should be. they are not db files.